toolkit/scripts/setuplkgtoolchain.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
```

Comment:
Might be worth to add `set -e` to catch errors faster and be intentional about ignoring the ones we want to ignore.
```suggestion
set -e

```